### PR TITLE
Docs: IDE tip wrt to javadsl / scaladsl #25312

### PR DESCRIPTION
**Pull request by sebastianharko**
#25312 

Initial version ...IMO it's short and to the point ... WDYT ?

### CODE REVIEW

**Comment by ktoso** [comment]
That's good, thanks!

Can we also add a section like "IDE tips" to https://doc.akka.io/docs/akka/current/additional/index.html and add it there with a screenshot in case people can't find it and link to it from here as well?

**Review by raboof — COMMENTED** [review]
I like the text! Indeed could be nice to move it somewhere more general, as it's useful beyond just the Streams API - though it might still nice to refer to it from here.

**Comment by sebastianharko** [comment]
@ktoso @raboof 

- Added an "IDE Tips" section to "Additional" ( with screenshot for IntelliJ ) 
- In the "Stream quickstart" page, the note is now much shorter and refs the user to "IDE Tips" page

File: akka-docs/src/main/paradox/additional/ide.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ 
+## Configure the auto-importer in IntelliJ / Eclipse 
+
+For a smooth development experience, when using an IDE such as Eclipse or IntelliJ, you can disable the auto-importer from suggesting `javadsl` imports when working in Scala, or viceversa.
+
+In IntelliJ, the auto-importer settings are under "Editor" / "General" / "Auto Import". Use a name mask such as `akka.stream.javadsl*` or `akka.stream.scaladsl*` or `*javadsl*` or `*scaladsl*` to indicate the DSL you want to exclude from import/completion. See screenshot below: 
+
+![idetip-auto-importer-solution.png](../images/idetip-auto-importer-solution.png)
+
+Eclipse users can configure this aspect of the IDE by going to "Window" / "Preferences" / "Java" / "Appearance" / "Type Filters".
```
Comment (sebastianharko) [inline akka-docs/src/main/paradox/additional/ide.md]: is it worth having a screenshot for Eclipse users ?

**Review by raboof — APPROVED** [review]
Really useful :+1:

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**